### PR TITLE
mitigate async reserved keyword

### PR DESCRIPTION
**Pull request by NickPancakes**
#24 fixed by renaming instances of "async" to "asynchronous".

### CODE REVIEW

**Comment by NickPancakes** [comment]
Looks like py37 isn't working on Travis at the moment.  
There's a workaround posted on travis-ci/travis-ci#9815 . I could add
`dist: xenial
sudo: true`
to get py37 to work on Travis if you're comfortable with those changes.

**Comment by mrname** [comment]
Sorry, didn't realize that Travis does not have official support yet, let's not worry about it until this is provided.

File: haralyzer/assets.py [hunk 1]
```diff
@@ -341,11 +341,11 @@ def filter_entries(self, request_type=None, content_type=None,
         return results
 
     def get_load_time(self, request_type=None, content_type=None,
-                      status_code=None, async=True):
+                      status_code=None, **kwargs):
```
Comment (mrname) [inline haralyzer/assets.py]: So sorry, I should have been more clear about this. What you originally did was perfect (using the keyword argument `asynchronous`, as this is what people should be doing moving forward. However, I was thinking we would use `kwargs` to catch the old `async` keyword, allowing both to be used. This way, we could eventually fully deprecate the old `async` keyword argument. Does that make sense?

Comment (NickPancakes) [reply haralyzer/assets.py]: Haha, yes. I'll give it a 3rd go later.

Comment (mrname) [reply haralyzer/assets.py]: Lol, so sorry. I'm happy to take over from here if it is getting annoying!

**Comment by NickPancakes** [comment]
Alright I think we're set. I did also test with py27 with async=False in test_get_load_time and it did in fact work as expected.

**Comment by mrname** [comment]
Thanks for humoring me @NickPancakes, looks good! Will try to cut a release today or tomorrow. Thanks for contributing!

**Comment by NickGlauberSmarsh** [comment]
Not a problem, thanks for the lib!

**Pull request merged**